clang/include/clang/Driver/Options.td

```diff
@@ -7036,19 +7036,22 @@ def fsycl_dead_args_optimization : Flag<["-"], "fsycl-dead-args-optimization">,
 def fno_sycl_dead_args_optimization : Flag<["-"], "fno-sycl-dead-args-optimization">,
   HelpText<"Disables elimination of DPC++ dead kernel arguments">;
 def fsycl_device_lib_EQ : CommaJoined<["-"], "fsycl-device-lib=">,
-  Flags<[NoXarchOption]>,
+  Flags<[Deprecated]>,
```

Comment:
@jinge90, there is also an associated test `clang/test/Driver/sycl-deprecated.cpp` can that test be updated?